docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1115,63 +1115,165 @@ For the purposes of our simulation, we made a few assumptions:
     women, not iron fortification of flour among children.
 
 5.  We assumed that all individuals covered by baseline coverage of iron
-    fortification have been covered for at least six months and therefore have
-    already achieved the full effect of the intervention.
+    fortification have been covered for at least six months, and therefore
+    either
 
-Therefore, the effect size of iron fortification on a simulant's hemoglobin
-level **in the baseline scenario** should be determined as follows:
+    a) They have already achieved the full age-dependent effect of the
+    intervention *if* they have been eating food for at least 6 months; that is,
+    they receive the full effect if they are at least one year old, or
+
+    b) If they are less than one year old, they have started or will start
+    eating fortified food as soon as they turn(ed) 6 months old, hence will
+    achieve the full age-dependent effect at exactly one year old.
+
+To summarize, we are trying to account for two factors that will decrease the
+reported effect size of iron fortification on hemoglobin in our simulants:
+
+1.  (**Age effect**) Children don't start eating (fortified) solid food until 6
```

Comment:
don't start eating _(much)_ (fortified) solid food

Reply:
I think saying "don't start eating *much* fortified solid food" would just muddle the description. I'm trying to clearly describe our algorithm to tell the SWEs what to build (and to remind the researchers what we decided to build), and our algorithm says children don't eat *any* fortified food until 6 months.  The discussion of how we arrived at this decision comes earlier in the section. If we don't think this approximation is close enough, then we should just change the function.